### PR TITLE
Fixed the blocked elements on the login page

### PR DESCRIPTION
**Pull request by EasyVector**
Dear developer:

Hello! I am the creator of issue #183, and I have fixed the blocked elements on the login page. I would appreciate it if you could kindly revise my code and leave me some advice. Thank you so much for your precious time!

Here is the screenshot after my changes:

copy

Thanks again! Have a nice day! :)

### CODE REVIEW

**Comment by franklinbill** [comment]
After I fixed it according to your repair method on Android studio, the bug was fixed. It's great! @yuhuitech

**Comment by dkanada** [comment]
I think the changes look fine, but can you recommit your changes without any formatting changes? Both commits have a diff that's quite hard to follow and there's no reason to make things less clear than they need to be.

**Comment by dkanada** [comment]
After testing the pull request on my phone I don't think I'll be able to merge this without changes. The code itself is fine but the interface for normal font sizes is also modified, and I don't particularly like how it looks on those screens now.
